app/src/main/java/io/homeassistant/companion/android/widgets/button/ButtonWidgetConfigureActivity.kt

```diff
@@ -75,6 +75,7 @@ class ButtonWidgetConfigureActivity : BaseWidgetConfigureActivity(), IconDialog.
     private lateinit var binding: WidgetButtonConfigureBinding
 
     private var requestLauncherSetup = false
+    private var requireAuthentication: Boolean = false
```

Comment:
The type can be inferred, you can remove `: Boolean`.

Reply:
OK